ui/src/alerting/components/CheckCards.tsx

```diff
@@ -27,17 +35,42 @@ const CheckCards: FunctionComponent<Props> = ({checks}) => {
   )
 }
 
-const EmptyChecksList: FunctionComponent = () => {
+interface EmptyProps {
+  showFirstTimeWidget: boolean
+}
+
+const EmptyChecksList: FunctionComponent<EmptyProps> = ({
+  showFirstTimeWidget,
+}) => {
+  if (showFirstTimeWidget) {
+    return (
+      <Panel
+        gradient={Gradients.GundamPilot}
+        size={ComponentSize.Large}
+        className="alerting-first-time"
+      >
+        <Panel.Body>
+          <h1>
+            Looks like it's your
+            <br />
+            first time here
+          </h1>
+          <h5>
+            Welcome to our new Monitoring & Alerting feature!
+            <br />
+            Try creating a <strong>Check</strong> to get started
+          </h5>
+        </Panel.Body>
+      </Panel>
+    )
+  }
+
   return (
     <EmptyState size={ComponentSize.Small} className="alert-column--empty">
       <EmptyState.Text
-        text="A Check  is a periodic query that the system performs against your time series data that will generate a status"
+        text="Looks like you have not created a Check  yet LINEBREAK LINEBREAK You will need one to be notified about LINEBREAK any changes in system status"
```

Comment:
What's up with this `LINEBREAK` stuff? Could markdown be passed instead? Or the text passed as multiple props?

Reply:
Glad you asked! The empty state component is overly strict and ends up shooting itself in the foot in the process. I am planning a small change in the next clockface update that should remove all this funk

Reply:
Ah ok, awesome